javascript/builtins/String.json

```
@@ -1824,9 +1824,16 @@
               "firefox_android": {
                 "version_added": "4"
               },
-              "ie": {
-                "version_added": "4"
-              },
+              "ie": [
+                {
+                  "version_added": "5.5"
+                },
+                {
+                  "version_added": "4",
```

The note here doesn't apply to later versions, so we can close this support statement when the next one begins.

```suggestion
                  "version_added": "4",
                  "version_removed": "5.5",
```

---

Agree. Fixed.